javatests/arcs/core/storage/CapabilitiesResolverTest.kt

```diff
@@ -38,71 +45,71 @@ class CapabilitiesResolverTest {
         val ramDiskCreator: Pair<Capabilities, StorageKeyCreator> = Capabilities.TiedToRuntime to
-            { storageKeyOptions, _ -> RamDiskStorageKey(storageKeyOptions.arcId.toString()) }
+            { storageKeyOptions -> RamDiskStorageKey(storageKeyOptions.arcId.toString()) }
         val resolver = CapabilitiesResolver(
             options,
             mutableMapOf(RAMDISK_DRIVER_PROTOCOL to ramDiskCreator)
         )
         assertThrows(IllegalArgumentException::class) {
-            resolver.createStorageKey(Capabilities.TiedToArc)
+            resolver.createStorageKey(Capabilities.TiedToArc, thingSchema, handleId)
         }
-        assertThat(resolver.createStorageKey(Capabilities.TiedToRuntime))
-            .isInstanceOf(RamDiskStorageKey::class.java)
+        verifyStorageKey((resolver.createStorageKey(Capabilities.TiedToRuntime, thingSchema, handleId)), RamDiskStorageKey::class.java)
     }
 
     @Test
     fun capabilitiesResolver_createsStorageKeys() {
         RamDisk.clear()
         DatabaseDriverProvider.configure(MockDatabaseManager(), mapOf<String, Schema>()::get)
-        val options = CapabilitiesResolver.StorageKeyOptions(ArcId.newForTest("test"))
+        val options = CapabilitiesResolver.CapabilitiesResolverOptions(ArcId.newForTest("test"))
         val resolver1 = CapabilitiesResolver(options)
         assertThat(resolver1.findStorageKeyProtocols(Capabilities.TiedToArc))
             .containsExactly(VOLATILE_DRIVER_PROTOCOL)
         assertThat(resolver1.findStorageKeyProtocols(Capabilities.TiedToRuntime))
             .containsExactly(RAMDISK_DRIVER_PROTOCOL)
         assertThat(resolver1.findStorageKeyProtocols(Capabilities.Persistent))
             .containsExactly(DATABASE_DRIVER_PROTOCOL)
-        assertThat(resolver1.createStorageKey(Capabilities.TiedToArc))
-            .isInstanceOf(VolatileStorageKey::class.java)
-        assertThat(resolver1.createStorageKey(Capabilities.TiedToRuntime))
-            .isInstanceOf(RamDiskStorageKey::class.java)
-        assertThat(resolver1.createStorageKey(Capabilities.Persistent, "abc012"))
-            .isInstanceOf(DatabaseStorageKey::class.java)
+        verifyStorageKey(resolver1.createStorageKey(Capabilities.TiedToArc, thingSchema, handleId), VolatileStorageKey::class.java)
+        verifyStorageKey(resolver1.createStorageKey(Capabilities.TiedToRuntime, thingSchema, handleId), RamDiskStorageKey::class.java)
+        verifyStorageKey(resolver1.createStorageKey(Capabilities.Persistent, thingSchema, handleId), DatabaseStorageKey::class.java)
 
         CapabilitiesResolver.reset()
         val resolver2 = CapabilitiesResolver(options)
-        assertThat(resolver2.createStorageKey(Capabilities.TiedToArc))
-            .isInstanceOf(VolatileStorageKey::class.java)
+        verifyStorageKey(resolver2.createStorageKey(Capabilities.TiedToArc, thingSchema, handleId), VolatileStorageKey::class.java)
```

Comment:
And here... I wonder -  did someone disable ktlint for javatests/?

Reply:
dunno  :( but fixed everything in this file.

Reply:
Thanks!